src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -378,6 +378,13 @@ public static bool TryText(FormattingInfo formatInfo, IRContext irContext, Formu
             var formatString = textFormatArgs.FormatArg;
             result = null;
 
+            // There is a difference between Windows 10 and 11 for French locale
+            // We fix the thousand separator here to be consistent 
+            if (culture.Name == "fr-FR" && culture.NumberFormat.NumberGroupSeparator == "\u00A0")

```

Comment:
can you do ignorecase string comparation?

Reply:
done